swift-sdk/Internal/InAppManager.swift

```diff
@@ -215,23 +243,23 @@ extension InAppManager : InAppSynchronizerDelegate {
         }
     }
     
-    private func scheduleMessages() {
+    private func scheduleNextMessage() {
         ITBDebug()
         
         scheduleQueue.async {
-            let waitTimeInterval = self.getWaitTimeInterval()
+            let waitTimeInterval = self.getInAppShowingWaitTimeInterval()
             if waitTimeInterval > 0 {
                 ITBDebug("Need to wait for: \(waitTimeInterval)")
                 self.scheduleQueue.asyncAfter(deadline: .now() + waitTimeInterval) {
-                    self.processMessages()
+                    self.processNextMessage()
                 }
             } else {
-                self.processMessages()
+                self.processNextMessage()
             }
         }
     }
     
-    private func processMessages() {
+    private func processNextMessage() {
```

Comment:
Why is is `nextMessage` now?

Reply:
It is the basically the same method just the name is changed. I think it makes the intention clearer. It process the next 'processable' message. It will not process all messages just the first processable message.